repos/rust-lang/annotate-snippets-rs.toml

```
@@ -9,4 +9,5 @@ wg-diagnostics = "write"
 
 [[branch-protections]]
 pattern = "master"
+ci-checks = ["ci"]
```

```suggestion
ci-checks = ["CI"]
```

Since the job has a name, I think the ci-check should respect that name. E.g. see https://github.com/rust-lang/crates-io-auth-action/blob/9dc75e2fccbefc31e71a232eaee19726b2c9b446/.github/workflows/ci.yml#L181 and https://github.com/rust-lang/team/blob/e9dc0fab73d8928993e794cba9530f0adf96b92e/repos/rust-lang/crates-io-auth-action.toml#L13